src/remoteplayer.h

```diff
@@ -110,9 +110,15 @@ class RemotePlayer : public Player
 
 	const CloudParams &getCloudParams() const { return m_cloud_params; }
 
-	bool checkModified() const { return m_dirty || inventory.checkModified(); }
+	u32 getLifetime() const { return (time(NULL) - m_oldtime); }
 
-	inline void setModified(const bool x) { m_dirty = x; }
+	u32 getIdletime() const { return (time(NULL) - m_newtime); }
```

Comment:
We have `porting::getTimeS()` for this, it provides a monotonic clock.
also the type should be `u64`